subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/notes/tasks/UpdateReleaseNotes.java

```diff
@@ -108,6 +108,6 @@ private String getVcsCommitTemplate(UpdateReleaseNotesTask task) {
     }
 
     public String getReleaseNotesUrl(UpdateReleaseNotesTask task, String branch) {
-        return  task.getGitHubUrl() + "/" + task.getGitHubRepository() + "/blob/" + branch + "/" + task.getProject().relativePath(task.getReleaseNotesFile());
+        return task.getGitHubUrl() + "/" + task.getGitHubRepository() + "/blob/" + branch + "/" + task.getProject().relativePath(task.getReleaseNotesFile()).replace('\\', '/');
```

Comment:
Cool! Do you mind extracting a utility method to some other class, for getting the relative path safely. For example:

```Java
String relativePath(Project project, File file)
```